src/bindings/glib/tests/CMakeLists.txt

```diff
@@ -14,9 +12,7 @@ macro (do_glib_test file)
 		COMPILE_DEFINITIONS HAVE_KDBCONFIG_H
 	)
 
-	set (GLIBS ${GELEKTRA_LIBRARY} ${GOBJECT_LIBRARIES})
-	tool_link_elektra (${name})
-	target_link_libraries (${name} ${GLIBS})
+	target_link_libraries (${name} PRIVATE ${GELEKTRA_LIBRARY})
```

Comment:
only to understand which parts require the new CMake, it is not documented above.